src/store/ui.js

```diff
@@ -173,6 +201,9 @@ export default {
   }
 }
 
+const isMosaicId = (str) =>
```

Comment:
MosaicID and NamespaceID is same length actually. 
maybe we should rename it to `isMosaicOrNamespaceId`

Reply:
Yes, good proposal, thanks. Changed in last commit